source/_integrations/whisper.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Whisper
+description: Connect and control your Whisper instance using the Wyoming Protocol integration
+ha_category:
+  - Voice
+ha_domain: whisper
+ha_release: '2023.5'
```

Comment:
Is it this integration release or wyoming release?
```suggestion
ha_release: '2023.6'
```

Reply:
wyoming, it follow the original integration